pkg/specgen/generate/pod_create.go

```diff
@@ -46,6 +46,13 @@ func createPodOptions(p *specgen.PodSpecGenerator) ([]libpod.PodCreateOption, er
 	if len(p.HostAdd) > 0 {
 		options = append(options, libpod.WithPodHosts(p.HostAdd))
 	}
+	if len(p.DNSServer) > 0 {
```

Comment:
Need to check for `--dns=none`

Reply:
that is already managed through `NetFlagsToNetOptions`